### PR TITLE
Adding Cucumber Test

### PR DESCRIPTION
**Pull request by techmaharaj**
Adding code for Cucumber test using Gradle.

### CODE REVIEW

**Review by vsukhin — APPROVED** [review]
looks cool!

**Comment by vsukhin** [comment]
@tkonieczny any commennts?

**Comment by techmaharaj** [comment]
Hey @vsukhin thanks for reviewing this. Bruno has seen and approved and I have to update the repo links in the blog post, so I will go ahead and merge this.

**Comment by vsukhin** [comment]
sure! go forward

**Comment by rangoo94** [comment]
One thing - should we have spaces in directory names or underscores? Probably worth to consider some convention 🙂

File: Cucumber Test Using Gradle/gradlew [hunk 1]
```diff
@@ -0,0 +1,249 @@
+#!/bin/sh
```
Comment (rangoo94) [inline Cucumber Test Using Gradle/gradlew:1]: Gradle Wrapper files should not be commited (`jar`/`gradlew` etc) and should be `.gitignore`d

**Pull request merged**